src/index.js

```diff
@@ -86,6 +95,19 @@ export default function(babel) {
               t.identifier('BigInt'), t.identifier(name)),
           args);
     }
+    if (dataViewMethods.has(name)) {
+      if (args.length !== 2 && args.length !== 3) {
+        throw path.buildCodeFrameError(
+            `${name} must have exactly two or three arguments`);
```

Comment:
This isn't quite correct: the two `...Get...` methods take 2 or 3 arguments, the two `...Set...` methods take 3 or 4. In all cases, `littleEndian` is optional.
I think in the interest of simplicity it's probably fine to just extend this check to accept 2-4 arguments for all methods.

Reply:
```suggestion
      if (2 <= args.length && args.length <= 4) {
        throw path.buildCodeFrameError(`${name}: incorrect number of arguments`);
```